### PR TITLE
NRPTI-12 NRO Inspections CSV import

### PR DESCRIPTION
**Pull request by BcGovNeal**
https://bcmines.atlassian.net/browse/NRPT-12

NRO Inspections CSV import.  Please use this [sample file](https://chat.pathfinder.gov.bc.ca/group/7yTf7q3mWbKBAn2qa?msg=JPivWJQyAGTPXph2i) to test.

### CODE REVIEW

File: api/src/importers/nro/base-record-utils.js [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+      throw Error('updateRecord - required existingRecord must be non-null.');
+    }
+
+    try {
+      // build update Obj, which needs to include the flavour record ids
+      const updateObj = { ...nrptiRecord, _id: existingRecord._id };
+
+      updateObj.updatedBy = (this.auth_payload && this.auth_payload.preferred_username) || '';
+      updateObj.dateUpdated = new Date();
+      updateObj.sourceDateUpdated = new Date();
```
Comment (marklise) [inline api/src/importers/nro/base-record-utils.js:101]: Should this not be a field populated from the datasource itself?

Comment (BcGovNeal) [reply api/src/importers/nro/base-record-utils.js]: Good catch!  That's actually not needed.  Removed in https://github.com/bcgov/NRPTI/pull/696/commits/a205ce08b7116d83ab23fb39ec28a1561c8e14f8

**Review by mtCarto — APPROVED** [review]
CSV import functions, records look correct 👍

**Review by MaxWardle — APPROVED** [review]
Data looks good after those small tweaks to  the  outcome description 👍

**Pull request merged**